src/test/components/AppLocalizationProvider.test.js

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { AppLocalizationProvider } from '../../components/app/AppLocalizationProvider';
+import { blankStore } from '../fixtures/stores';
+import { Provider } from 'react-redux';
+import { requestL10n, receiveL10n } from '../../actions/l10n';
+import { isL10nFetching, getLocalizations } from '../../selectors/app';
+import { lazilyParsedBundles } from 'firefox-profiler/app-logic/l10n-bundling';
+import { ReactLocalization } from '@fluent/react';
+
+describe('AppLocalizationProvider', () => {
+  beforeEach(function() {
+    const mockFetch = Promise.resolve({
+      text: () => Promise.resolve('resolved'),
+    });
+    global.fetch = jest.fn().mockImplementation(() => mockFetch);
```

Comment:
I was having the following flow error when I used `window.fetch`,
![image](https://user-images.githubusercontent.com/37832457/103707845-9c143680-4fd9-11eb-9542-8afb6dffdc6a.png)

Reply:
This is giving an error because the mockImplementation is actually waiting a `Response` type from us. `global` is not covered by flow, that's why it's not giving us any error. It should work if you do something like this:
```js
const response = (({
  ok: true,
  status: 200,
  headers: {
    get: () => 'text/plain',
  },
  text: () => Promise.resolve('resolved'),
}: any): Response);
window.fetch = jest.fn().mockResolvedValue(response);
```